modules/ip/api/ip-routes.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+
+function genMAC(){
+    var hexDigits = "0123456789ABCDEF";
+    var macAddress = "";
+    for (var i = 0; i < 6; i++) {
+        macAddress+=hexDigits.charAt(Math.round(Math.random() * 15));
+        macAddress+=hexDigits.charAt(Math.round(Math.random() * 15));
+        if (i != 5) macAddress += ":";
+    }
+    return macAddress;
+}
+
+    app.get("/ip/", (req: Request, res: Response) => {
```

Comment:
Thank you for this, we no longer directly edit swagger files instead we use OpenAPI comments in the api route file, please can you update this.

Reply:
We also like to give the user the ability to define how many they would like in return, please can you add a qty param to the url and use the util function getQtyFromRequest to get the qty

Reply:
added OpenAPI comments in the api route file, also added qty parameter in ip and macusing function  getQtyFromRequest to get the qty